docs/csharp/quick-starts/introduction-to-classes.md

```diff
@@ -0,0 +1,177 @@
+ms.devlang: csharp
+---
+# Introduction to classes
+
+This lesson assumes installed [.NET Core](http://dot.net/core) and [Visual Studio Code](https://code.visualstudio.com/), or [Visual Studio](https://www.visualstudio.com/) on Mac or Windows.
+
+## Create your application
+
+Create a directory named **classes**. You'll build your application there. Change to that directory and type `dotnet new console`. This command creates your application. Open **Program.cs**. It should look like this:
+
+```csharp
+using System;
+
+namespace classes
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hello World!");
+        }
+    }
+}
+```
+
+In this quick start, you're going to create new types that represent a bank account. Typically developers define each class in a different text file. That makes it easier to manage as a program grows in size.  Create a new file named **BankAccount.cs** in the **classes** directory. 
+
+This file will contain the deefinition of a ***bank account***. Object Oriented programming organizes code by creating types in the form of ***classes***. These classes contain the code that represent a concept. The `BankAccount` class represents the responsibilities and properties of a bank account. In this quick start, the bank account supports this behavior:
+
+1. It has a 10-digit number that uniquely identifies the bank account.
+1. It has a string that stores the name or names of the owners.
+1. The balance can be retrieved.
+1. It accepts deposits.
+1. It accepts withdrawals.
+
+## Define the bank account type
+
+You can start by creating the basics of a class that defines that behavior. It would look like this:
+
+```csharp
+using System;
+
+namespace classes
+{
+    public class BankAccount
+    {
+        public string Number { get; }
+        public string Owner { get; set; }
+        public decimal Balance { get; }
+
+        public void MakeDeposit(decimal amount, DateTime date, string note)
+        {
+        }
+
+        public void MakeWithdrawal(decimal amount, DateTime date, string payee, string note)
+        {
+        }
+    }
+}
+```
+
+Before going on, let's take a look at what you've built.  The `namespace` declaration provides a way to logically organize your code. This quick start is relatively small, so you'll put all the code in one namespace. 
+
+The `public class BankAccount` defines the class, or type, you are creating. Everything inside the `{` and `}` that follows the class declaration defines the behavior of the class. There are five ***members*** of the `BankAccount` class. The first three are ***properties***. Properties are data elements, but can have code that enforces validation or other rules. The last two are ***methods***. Methods are blocks of code that peform a single function. Reading the names of each of the members should provide enough information for you or another developer to understand what the class does.
```

Comment:
delete "The" (the first word of the paragraph)
data elements, but --> data elements, and they can also